Source/santad/EventProviders/SNTEndpointSecurityFileAccessAuthorizer.mm

```diff
@@ -310,6 +310,48 @@ - (FileAccessPolicyDecision)specialCaseForPolicy:(std::shared_ptr<WatchItemPolic
   return FileAccessPolicyDecision::kNoPolicy;
 }
 
+/// An An `es_process_t` must match all criteria within the given
+/// WatchItemPolicy::Process to be considered a match.
+- (bool)policyProcess:(const WatchItemPolicy::Process &)policyProc
+     matchesESProcess:(const es_process_t *)esProc {
+  if (esProc->codesigning_flags & CS_SIGNED) {
```

Comment:
Shouldn't you also be checking for CS_VALID?

Reply:
The `CS_VALID` check is handled prior to this method being called. This method is used for individually checking each of the configured process exceptions while the check for a valid code signature is more broad and applies whether or not process exceptions exist.

This does look confusing though and I'll add a comment.